database/Models/index.js

```diff
@@ -7,22 +7,19 @@ const sequelize = new Sequelize({
   database: 'sdc',
   password: 'test',
   port: 5432,
-  host: 'localhost'
+  host: 'localhost',
+  logging: false
 });
 
 const models = [
   require('./photos'),
-  //add other url model here
+  require('./rooms')
```

Comment:
This is interesting how you are using require. Are you storing your files in this array models?

Reply:
yeah the `require` statement returns what is exported from each model file, which is a function